src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+html body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  --text-active: hsl(193, 100%, 43%);
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__item {
+  display: block;
+  margin-left: 20px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
```

Comment:
I suggest separating the `reset` styles from other. Try writing it above as:
```
ul {
  padding: 0;
  margin: 0;
  list-style: none;
}
```
Do the same for every reset styles